downstream/modules/platform/con-aap-upgrade-prereq.adoc

```diff
@@ -0,0 +1,147 @@
+
+.Creating non-root user with sudo privileges
+Prior to your upgrade of {PlatformNameShort}, it is recommended to create a non-root user with sudo privileges for the deployment process. This user is used for:
+
+* SSH connectivity.
+* Passwordless authentication during installation.
+* Privilege escalation (sudo) permissions.
+
+The following examples uses `ansible` to name this user. On all nodes used in the upgraded {PlatformNameShort} cluster, create a non-root user named `ansible` and generate an ssh key:
+
+. Create a non-root user:
++
+----
+# useradd ansible
+----
+. Set a password for your user:
++
+----
+# passwd ansible <1>
+Changing password for ansible.
+Old Password:
+New Password:
+Retype New Password:
+----
+<1> Replace `ansible` with the non-root user from step 1, if using a different name
+. Generate an `ssh` key as the user:
++
+----
+$ ssh-keygen -t rsa
+----
+. Disable password requirements when using `sudo`:
++
+----
+# echo "ansible ALL=(ALL) NOPASSWD:ALL" | sudo tee -a /etc/sudoers.d/ansible
+----
+
+.Copying SSH keys to all nodes
+WIth the `ansible` user created, copy the `ssh` key to all the nodes used in the upgraded {PlatformNameShort} cluster. This ensures that when the {PlatformNameShort} installation runs, it can `ssh` to all the nodes without a password:
+----
+$ ssh-copy-id ansible@node-1.example.com
+----
+NOTE: If running within a cloud provider, you may need to instead create an `~/.ssh/authorized_keys` file containing the public key for the `ansible` user on all your nodes and set the permissions to the `authorized_keys` file to only the owner (`ansible`) having read and write access (permissions 600).
+
+.Configuring firewall settings
+Configure the firewall settings on all the nodes used in the upgraded {PlatformNameShort} cluster to permit access to the appropriate services and ports for a successful {PlatformNameShort} upgrade. For Red Hat Enterprise Linux 8 or later, enable the `firewalld` daemon to enable the access needed for all nodes:
+
+. Install the `firewalld` package:
++
+----
+# dnf install firewalld --assumeyes
+----
+. Start the `firewalld` service:
++
+----
+# systemctl start firewalld
+----
+. Enable the `firewalld` service:
++
+----
+# systemctl enable --now firewalld
+----
+
+
+== {PlatformNameShort} configuration requirements
+
+.Configuring firewall settings for execution and hop nodes
+
+After upgrading your {PlatformName} instance, add the automation mesh port on the mesh nodes (execution and hop nodes) to enable automation mesh functionality. The default port used for the mesh networks on all the nodes is set to `27199/tcp`, however it can also be configured to use a different port by specifying `receptor_listener_port` as each node’s variable within your `inventory` file.
```

Comment:
Suggest rewording second sentence to active voice: The default port used for the mesh networks on all nodes is `27199/tcp`. You can configure the mesh network to use a different port by specifying `recptor_listener_port` as the variable for each node within your inventory file.